docs/source/ProgrammingGuide/View.md

```diff
@@ -80,7 +80,7 @@ While it is in principle possible to have Kokkos Views of arbitrary objects, Kok
 * `T`'s assignment operators as well as its default constructor and deconstructor must be marked with the `KOKKOS_INLINE_FUNCTION` or `KOKKOS_FUNCTION` macro.
 
 All restrictions but the first come from the requirement that `View<T*>` work with every execution and memory space,
-including those that use CUDA. The constructor of `View<T*>` does not just allocate memory; it also initializes the allocation by iterating over it using a [`Kokkos::parallel_for`](../API/core/parallel-dispatch/parallel_for), and invoking `T`'s default constructor for each entry. If the View's execution space is `Cuda`, then that means `T`'s default constructor needs to be correct to call on device. Keep in mind that the semantics of the resulting `View` are a combination of the `Views 'view'` semantics and the behavior of the element type.
+including those that use CUDA. The constructor of `View<T*>` does not just allocate memory; it also initializes the allocation with `T`'s default value for each entry. If the View's execution space is `Cuda`, then that means `T`'s default constructor needs to be correct to call on device. Keep in mind that the semantics of the resulting `View` are a combination of the `Views 'view'` semantics and the behavior of the element type.
```

Comment:
Would it be worthwile getting rid of the `Cuda` and `device` here?  e.g.:
`T`'s default constructor needs to be correct to call on the `ExecutionSpace` associated to the `MemorySpace` of the `View`.

Reply:
Sounds good to me.